main.css

```diff
@@ -0,0 +1,293 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@900&display=swap');
+
+body {
+  font-family: Poppins, sans-serif;
+  margin: 0;
+}
+
+.color-white-style {
+  color: white;
+}
+
+header {
+  background-color: #1c1a19;
+  background-image: url("./Images/illustration-header-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 125px;
+  background-position: right bottom;
+}
+
```

Comment:
. Add space between the header and the title

![Screen Shot 2022-02-14 at 16 15 56](https://user-images.githubusercontent.com/38294716/153890030-5368a379-494f-4c8f-acb8-aff36d401b50.png)

Reply:
- [ ] Kindly give reasons why you are requesting this change 👍🏿

Reply:
Kindly can you add space between the header and the title as shown in the template